activestate.yaml

```diff
@@ -39,7 +39,9 @@ scripts:
     value: |
       set -e
       $constants.SET_ENV
-      go build -tags "$GO_BUILD_TAGS" -o $constants.BUILD_TARGET $constants.CLI_BUILDFLAGS $constants.CLI_PKGS
+      export GOARCH=${1:-amd64}
```

Comment:
What's the dash for?

Reply:
It expands to "amd64" as a default value in case `$1` is empty or unset: https://i.stack.imgur.com/T2Fp8.png